functions/signals/ks_main.php

```diff
@@ -68,6 +71,7 @@ public static function showDescription()
 	 */
 	public static function generateImage($height)
```

Comment:
The variable name of $height needs to be changed. It describes the vertical translation of the signal here.